src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/js/fuse_search.js

```diff
@@ -80,10 +80,20 @@ require(["fuse"], function (Fuse) {
     return text.replace(regex, '<span class="highlight">$1</span>');
   }
 
+  const pathParts = window.location.pathname.split("/");
+  let basePath;
+
+  // Check if the path contains a version directory
+  if (pathParts.includes("version")) {
+    basePath = pathParts.slice(0, 3).join("/"); // Extract up to /version/{version}
+  } else {
+    basePath = ""; // If there's no version, just use the base path
+  }
```

Comment:
This is tricky. What if a library has a folder named "version" with some RST files inside?

Reply:
yes, rewrite the code and now we are getting the relative path to search.json file using {{pathto}} jinja in html and the relative path to the root using data-content root from html header